src/labels/CustomLabels.labels

```diff
@@ -1896,6 +1896,30 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>Validate Recurring Donations data if it is valid install period</shortDescription>
         <value>{0} is not a supported value for Installment Period. Select Daily, Weekly, 1st and 15th, Monthly, or Yearly.</value>
     </labels>
+    <labels>
+        <fullName>RD2_NumberInstallmentsRequiredWhenFixed</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Require Planned Installments field on the RD when Recurring Type is Fixed</shortDescription>
+        <value>Number of Planned Installments is required when Recurring Type is Fixed.</value>
+    </labels>
+    <labels>
+        <fullName>RD2_NumberInstallmentsNotAllowedWhenOpen</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Disallow a value in Planned Installments on the RD for Recurring Type is Open</shortDescription>
+        <value>Number of Planned Installments is for Fixed Recurring Donations only. Remove the value in Number of Planned Installments or change the Recurring Type to Fixed.</value>
+    </labels>
+    <labels>
+        <fullName>RD2_NumberPlannedInstallmentsCannotBeLessThanPaid</fullName>
```

Comment:
@force2b , can we change this label name to `RD2_NumberInstallmentsCannotBeLessThanPaid` to follow other two custom labels name (`Planned` is not not required in the name)?
@meena-sfdc ^